pastas/timeseries_utils.py

```diff
@@ -518,3 +519,35 @@ def pandas_equidistant_asfreq(series: Series, freq: str) -> Series:
         .squeeze()
     )
     return spandas
+
+
+def resample(
+    series: Series, freq: str, closed: str = "right", label: str = "right", **kwargs
+) -> Resampler:
+    """Wrapper around the pandas resample function with logical some Pastas defaults.
+       In Pastas, we assume the timestamp is at the end of the period that
+       belongs to each measurement. For more information on this read the
+       example notebook on preprocessing time series.
+
+    Parameters
+    ----------
+    series : Series
+        time series
+    freq : str
+        frequency string
+    closed: str, default 'right'
+        which side/end of bin interval is closed
+    label: str, default 'right'
+        which bin edge label to label bucket with
```

Comment:
bucket = bins?

Reply:
Fair enough. I was just very slightly confused by bin/bucket in the same sentence :P. Fine to keep it this way.